predicators/envs/pybullet_cover.py

```diff
@@ -193,6 +193,17 @@ def _reset_state(self, state: State) -> None:
                 assert self._held_obj_id == block_id
                 self._create_grasp_constraint()
 
+        # For any blocks not involved, put them out of view.
+        h = self._obj_len_hgt
+        oov_x, oov_y = self._out_of_view_xy
+        for i in range(len(block_objs), len(self._block_ids)):
```

Comment:
So `block_objs` are the blocks involved while the others are not? Why are the unit tests creating environments with >2 blocks (maybe its intentional)?

Reply:
it's actually the opposite: unit tests are sometimes creating environments with 1 block, and the other pybullet block object was then left where ever it was in the previous task